deployments/local/dev/docker-compose-keycloakx.yml

```diff
@@ -78,7 +79,7 @@ services:
 #      - "--spi-well-known-openid-configuration-include-client-scopes=true"
 # Workaround to allow logouts via old Keycloak Admin-Console
 # see: org.keycloak.protocol.oidc.endpoints.LogoutEndpoint.logout(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)
-      - "--spi-login-protocol-openid-connect-legacy-logout-redirect-uri=true"
+      - "--spi-login-protocol-openid-connect-legacy-logout-redirect-uri=false"
```

Comment:
I don't think that we need the legacy support anymore, so we can safely turn it "off".